### DISCUSSION TITLE
关于计算器的结果四舍五入

### DISCUSSION THREAD

**Discussion by bsnmldb** [liusy58/NJU_AdvancedProgramming_SP22#28] [Q&A] [answered]
如果是四舍五入的话显然2/3是0.7，1/3是0.3
那么允许10*（5.5/10）的结果为6吗

**Comment by RicoloveFeng** [comment, marked as answer]
四舍五入针对的是第二位小数，只一位小数当然是直接输出啊。注意中间格式是不限定一位小数的。

**Reply by bsnmldb** [reply]
我的意思是对于10*（5.5/10）先算5.5/10结果是0.55但返回的是0.6（递归），然后10*0.6得到6.0
这样不允许吗

**Reply by RicoloveFeng** [reply]
不允许中途四舍五入
